p2p/providers/peerstore_provider/peerstore_provider.go

```diff
@@ -1,6 +1,6 @@
 package peerstore_provider
 
-//go:generate mockgen -source=$GOFILE -destination=../../types/mocks/peerstore_provider_mock.go -package=mock_types github.com/pokt-network/pocket/p2p/types PeerstoreProvider
+//go:generate mockgen -destination=../../types/mocks/peerstore_provider_mock.go -package=mock_types github.com/pokt-network/pocket/p2p/providers/peerstore_provider PeerstoreProvider
```

Comment:
### Rational behind 89548d999ab1ce30cfc6df5d9be425ef95cc780e

@reviewer `mockgen` source mode seems to have trouble with embedded interfaces:

![image](https://user-images.githubusercontent.com/600733/228819163-accbffbe-aaaa-43a1-8ed6-bc75752026e4.png)

(see: https://github.com/golang/mock/issues/10)

Reply:
I think calling things out explicitly is even better